massa-execution-worker/src/execution.rs

```diff
@@ -305,6 +305,9 @@ impl ExecutionState {
             OperationType::Transaction { .. } => {
                 self.execute_transaction_op(&operation.content.op, sender_addr)
             }
+            OperationType::Denunciation { .. } => {
+                self.execute_denunciation_op(&operation.content.op, sender_addr, block_credits)
```

Comment:
To be clean, I would pass a mutable reference to new_block_credits, and move the "*block_credits = new_block_credits;" line after the call to execution

Reply:
done